client/my-sites/sidebar-unified/index.jsx

```diff
@@ -71,17 +96,27 @@ export const MySitesSidebarUnified = ( { path } ) => {
 								link={ item.url }
 								selected={ isSelected }
 								sidebarCollapsed={ sidebarIsCollapsed }
+								continueInCalypso={ continueInCalypso }
 								{ ...item }
 							/>
 						);
 					}
 
 					return (
-						<MySitesSidebarUnifiedItem key={ item.slug } selected={ isSelected } { ...item } />
+						<MySitesSidebarUnifiedItem
+							key={ item.slug }
+							selected={ isSelected }
+							continueInCalypso={ continueInCalypso }
+							{ ...item }
+						/>
 					);
 				} ) }
 				<CollapseSidebar key="collapse" title="Collapse menu" icon="dashicons-admin-collapse" />
 			</Sidebar>
+			<ExternalLinkDialog
+				isVisible={ showDialog }
+				closeModalHandler={ ( openUrl ) => closeModalHandler( openUrl ) }
```

Comment:
```suggestion
				closeModalHandler={ closeModalHandler }
```

Can we omit the extra function here?

Reply:
That's true 4832fa9!